src/stories/components/AccountMenu/AccountMenu.js

```diff
@@ -0,0 +1,117 @@
+import './styles.scss';
+
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { NavLink, useHistory } from 'react-router-dom';
+import { AuthContext } from '../../../Auth';
+import database from '../../../services/firestore-service';
+
+/**
+ * Functional Component for Account Menu
+ *
+ * @component
+ * @return {object} (
+ *  <AccountMenu />
+ * )
+*/
+
+export const AccountMenu = () => {
+  const { logout, currentUser } = useContext(AuthContext);
+  const [profile, setProfile] = useState(null);
+  const history = useHistory();
+
+  useEffect(() => {
+    if (currentUser) {
+      // setError('');
```

Comment:
ah cool, so we'll add a setError at some point to show the user whatever error firebase auth spits back. word.